### ISSUE TITLE
need support for setting global class prefix

### ISSUE THREAD

**Issue by aflext** [kazupon/vue-validator#254]
I need reset all the default classes like `valid` `invalid` `touched` and so on to have a specific prefix like `project-foo-` so i can style all of the validation elements with less like

```
project-foo {
    &-valid{
        border: 1px solid grey;
    };
    &-invalid{
       border: 1px solid red;
    };
}
```

What i expected is that vue-validator supports syntax like 

```
import Vue from 'vue';
import VueValidator from 'vue-validator';

// i want this
VueValidator.setGlobal('classPrefix', 'project-foo-');

Vuex.use(VueValidator);
```

**Comment by franciscolourenco** [comment]
I would suggest a way to globally customise classes further. For example:

```
VueValidator.defaults.classes.invalid = 'error'
VueValidator.defaults.classes.valid = 'good'
```

**Comment by kazupon** [comment]
released v3.0.0-alpha.1
https://github.com/vuejs/vue-validator/releases/tag/v3.0.0-alpha.1

We can use the 'Vue.config.validator.classes'.
See the [example](https://github.com/vuejs/vue-validator/blob/dev/examples/classes/global/index.html)
